python/pyspark/sql/tests/test_binding_parameters.py

```diff
@@ -0,0 +1,83 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.sql import SparkSession
+from pyspark.testing.sqlutils import ReusedSQLTestCase
+
+class BindingParametersTests(ReusedSQLTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(BindingParametersTests, cls).setUpClass()
+        cls.spark = SparkSession.builder.getOrCreate()
+        assert cls.spark is not None
+        assert cls.spark._jvm.SparkSession.getDefaultSession().isDefined()
+
+    def test_wrapping_plan_in_limit_node(self):
```

Comment:
Any reason why we need a PySpark specific test for this?

Reply:
`spark.sql` has no difference between Scala Spark and Python Spark, otherwise we will need to duplicate a lot of tests in pyspark.